docs/best-practices/background-jobs.md

```diff
@@ -69,13 +69,50 @@ If you require a background task to communicate with the calling task to indicat
+
+How to write an Azure Funtion 
+
+* You'll need to create an App Function Service, or you may try it for free if you don't have an Azure account yet. Then, you have these options: 
+ * Code functions directly from the portal 
+ * Use Visual Studio with tools for Azure Functions 
+ * Use Visual Studio Code and a Git repo for Continuous Delivery 
+
+Azure Functions can be coded in different languages (including as of today: JavaScript, C#, F#, as well as scripting options such as Python, PHP, Bash, Batch, and PowerShell). Azure Functions run within the context of a container Azure Function App. This allows them to access environment variables and share information, such as connection strings, with its parent Azure Function App. The job has access to the unique identifier of the machine that is running it. The connection string named **AzureWebJobsStorage** provides access to Azure storage queues, blobs, and tables for application data, and access to Service Bus for messaging and communication. The connection string named **AzureWebJobsDashboard** provides access to the job action log files. 
+
+Azure Functions have the following characteristics: 
+
+* **Security**: anonumous access or Authentication/Authorization thru various providers can be setup. Also, CORS settings are ready to be configured at App Settings 
+
+* **Monitoring**: Azure Functions activity can be monitored onva per-function basis 
+
+* **Deployment**: Git can be used as CD source origin, right after playground phase is completed and robust source control is required for better maintenance. 
+
+### Considerations 
+
+*	Type of tasks: short lived, specialized, low complexity and few/simple dependencies. 
+
+*	Not designed for long running workflow 
+
+*	On Consuption Plan there's a max timeout of 5 minutes to avoid errors to become costly. [Check this disussion](https://github.com/Azure/azure-webjobs-sdk-script/issues/18) and [this other one as well](https://github.com/Azure/Azure-Functions/issues/75). 
+
+*	Fault isolation will vary depending if runs on Consumption or App Service Plan. The first one auto scales in terms of assigned resources while the latter is constrained. This means that on Consumption Plan, a Function that eats resources intensively, can become a wallet drainer if gets out of control (*), but won't affect performance for others as resources are facilitated on demand. Opposedly, if running on App Service Plan, given that all Functions on a single App are bounded by predetermined hardware constraints, a resource intensive Function will affect others running in parallel. 
+
+(*) A Fuction running on Consuption Plan is topped at 5 minutes before forced expiration. 
+ 
+### More information 
+
+* [Azure Functions recommended resources](https://docs.microsoft.com/en-us/azure/azure-functions/functions-best-practices) extends over best practices for Azure Functions. 
```

Comment:
We should use relative URLs for content on Docs.
`/azure/azure-functions/functions-best-practices`